src/utils/beacon/useHandleBeaconMessage.tsx

```diff
@@ -194,6 +194,20 @@ export const partialOperationToOperation = (
         return { type: "undelegation", sender: signer.address };
       }
     }
+    case TezosOperationType.ORIGINATION: {
+      const { script } = partialOperation;
+      const { code, storage } = script as unknown as {
```

Comment:
why `as unknown`?

Reply:
It is a way of typecasting, considering `script` is defined as "string" in `TezosOriginationOperation`.
there is no way to cast it directly unless converting it to `unknown` first 

```
export interface TezosOriginationOperation extends TezosBaseOperation {
    ...
    script: string;
}
```

Reply:
I think we should update our type definition instead so that code can be either MichelsonJson[] | string, same as the storage

Reply:
this type comes from 3rd party lib. I'm not sure how we can affect that

Reply:
`script` comes in `message` as a deserialised structure, not as a `string`, despite `TezosOriginationOperation['script']: string`